docs/integration.md

```diff
@@ -0,0 +1,248 @@
+# [Integration Guide](#integration-guide)
+The basis of this document is to chowcase the process of integrating `bridget` into an existing web delivery. The end-goal of this module is adding bridging support to the website when browsed through the context of a mobile app via a WebView.
+
+## [Prerequisites](#prerequisites)
+
+This integration guide assumes you already have a web delivery that serves content & a mobile app is being setup for that  particular web delivery.
```

Comment:
I would split this in bullet points, so that it is very clear. Also marking it somehow "your web-site", "our app"... maybe yours and ours are not right words, but still, something that is in the hands of the integrator, and something that we provide as a product.